include/openthread-types.h

```diff
@@ -243,7 +241,11 @@ typedef struct otExtAddress
 /**
  * This structure represents an IPv6 address.
  */
+#ifdef _WIN32
+OT_TOOL_PACKED_BEGIN typedef struct otIp6Address
```

Comment:
There has got to be a better way. Perhaps if we define the struct first, and then define the typedef afterward?